model-optimizer/mo/ops/pooling.py

```diff
@@ -105,14 +117,26 @@ def pool_infer(node: Node):
             node['window'] = np.zeros(len(input_shape), dtype=np.int64)
             node.window[node.spatial_dims] = input_spatial_shape
 
+        if not node.has_valid('dilation'):
+            node['dilation'] = np.ones(len(input_shape))
+
+        if not node.has_valid('axis'):
+            node['axis'] = 0
+
+        if not node.has_valid('index_element_type'):
+            node['index_element_type'] = np.int64
+
         window_spatial_shape = node.window[node.spatial_dims]
         stride_spatial = node.stride[node.spatial_dims]
+        dilation_spatial = node.dilation[node.spatial_dims]
         assert any(stride_spatial), 'Stride can not be zero in node {}'.format(node.id)
 
         if node.has_valid('auto_pad') and node.auto_pad != 'explicit':
-            node.pad_spatial_shape, node.output_spatial_shape = tf_window_op_pad_infer(input_spatial_shape,
-                                                                                       window_spatial_shape,
-                                                                                       stride_spatial, node.auto_pad)
+            node.pad_spatial_shape, node.output_spatial_shape = tf_window_op_pad_infer(input=input_spatial_shape,
+                                                                                       window=window_spatial_shape,
+                                                                                       stride=stride_spatial,
+                                                                                       auto_pad=node.auto_pad,
+                                                                                       dilation=dilation_spatial)
```

Comment:
soft_get

Reply:
Setting the values of attributes, no need to use `soft_get`